README.md

```diff
@@ -32,16 +32,22 @@ permissions:
+| Name                   | Description                                                                                                                                                                                                              | Required | Default        |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- | -------------- |
+| `token`                | Coverage upload token generated when project is created on codelyze.com                                                                                                                                                  | yes      |                |
+| `path`                 | Relative path to the coverage file                                                                                                                                                                                       | yes      |                |
+| `gh-token`             | `GITHUB_TOKEN` (permissions `contents: write` and `pull-requests: write`) or a `repo` scoped [Personal Access Token (PAT)](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token). | no       | `GITHUB_TOKEN` |
+| `annotation`           | Add annotations to uncovered hunks                                                                                                                                                                                       | no       | true           |
+| `threshold`            | The minimum coverage allowed. If coverage is lower than this, the action will fail                                                                                                                                       | no       | 0              |
+| `difference-threshold` | The minimum total difference allowed between the current commit and the reference one                                                                                                                                    | no       | 0              |
+| `patch-threshold`      | the minimum value for patch coverage: the minimum ratio of new lines that are not covered.                                                                                                                               | no       | 0              |
+| `empty-patch`          | If true don't even add a commit status for a patch that has no changes.                                                                                                                                                  | no       | false          |
```

Comment:
I feel that `empty-patch: true` would mean to "add a commit status for a patch that has no changes", which is the reverse of the current definition. Maybe rename it as `skip-empty-patch`?

```suggestion
| `skip-empty-patch`          | If true don't even add a commit status for a patch that has no changes.                                                                                                                                                  | no       | false          |
```

Maybe I'm misunderstanding the purpose of this option.

Reply:
applied change